lib/pbench/test/unit/agent/test_utils.py

```diff
@@ -29,20 +29,24 @@ def test_constructor(self):
             bs = OurServer(arg1, arg2)
             assert bs.pid_file is None
             assert bs.host == "localhost"
-            assert bs.port == 4242
+            assert bs.port == OurServer.def_port
             assert bs.bind_host == bs.host
             assert bs.bind_port == bs.port
-            assert repr(bs) == "forty-two - localhost:4242"
+            assert repr(bs) == f"forty-two - localhost:{OurServer.def_port:d}"
```

Comment:
Since you made the port value dynamic, do you want to make the server name and/or the hostname dynamic?

Alternatively, if this is only used for debugging, do we really want to be testing it at all?

Reply:
I don't think so.  Using the `OurServer.def_port` class variable is fine because to avoid further perpetuating a constant, but `localhost` is so common that it doesn't gain us much making it dynamic.